numpy/distutils/misc_util.py

```diff
@@ -318,6 +318,7 @@ def make_temp_file(suffix='', prefix='', text=True):
 # Hooks for colored terminal output.
 # See also https://web.archive.org/web/20100314204946/http://www.livinglogic.de/Python/ansistyle
 def terminal_has_colors():
+    # Can this be cleaned up?
```

Comment:
this code goes back to at least 2005, I suspect it's fine now but I don't have a way to check.